cmd/neofs-cli/modules/control/shards_set_mode.go

```diff
@@ -39,12 +44,14 @@ func initControlSetShardModeCmd() {
 	flags.String(controlRPC, controlRPCDefault, controlRPCUsage)
 	flags.StringSlice(shardIDFlag, nil, "List of shard IDs in base58 encoding")
 	flags.Bool(shardAllFlag, false, "Process all shards")
+
+	modes := make([]string, 0, len(mShardModes))
+	for strMode := range mShardModes {
+		modes = append(modes, "'"+strMode+"'")
+	}
+
 	flags.String(shardModeFlag, "",
-		fmt.Sprintf("New shard mode keyword ('%s', '%s', '%s')",
-			shardModeReadWrite,
-			shardModeReadOnly,
-			shardModeDegraded,
```

Comment:
BTW what about removing `degraded` from help? It is not really intended to be used (`degraded-read-only` should) as it has it's has it's own problems. So we still support it but not suggest it to a user.

Reply:
Done